apps/dashboard/src/components/workflow-editor/steps/configure-step-form.tsx

```diff
@@ -188,12 +189,14 @@ export const ConfigureStepForm = (props: ConfigureStepFormProps) => {
       });
     });
 
-    // Set new errors from stepIssues
-    Object.entries(stepIssues).forEach(([key, value]) => {
-      // @ts-expect-error - dynamic key
-      form.setError(`controlValues.${key}`, { message: value });
-    });
-  }, [form, step]);
+    if (!hideValidationErrorsOnFirstRender) {
```

Comment:
Do not show step issues right after a step creation. But keep them in the FE memory, just don't hydrate RHF!